hera_mc/autocorrelations.py

```diff
@@ -26,9 +26,10 @@
 measurement_func_dict = {"median": np.median}
 
 
-def _get_autos_from_redis(redishost=DEFAULT_REDIS_ADDRESS):
-
-    redis_pool = redis.ConnectionPool(host=DEFAULT_REDIS_ADDRESS)
+def _get_autos_from_redis(redishost=None):
+    if redishost is None:
+        redishost = DEFAULT_REDIS_ADDRESS
+    redis_pool = redis.ConnectionPool(host=redishost)
```

Comment:
I changed that in consultation with Matt.  He had _get_autos_from_redis(redishost=DEFAULT_REDIS_ADDRESS):
   ...
   redis_pool = redis.ConnectionPool(host=DEFAULT_REDIS_ADDRESS)

i.e. he never used 'redishost' (which isn't good and also I needed it).  He wanted _get_autos_from_redis() to go to the right default, hence the None thing.

Reply:
ok, let's set `redishost=DEFAULT_REDIS_ADDRESS` in the function signature then, rather than setting it to `None` and then handling the `None` in extra lines.

Reply:
Done